package.json

```diff
@@ -107,6 +107,7 @@
     "test-prod-build": "yarn test-build-prod",
     "test-build": "cross-env NODE_ENV=development jest --config ./scripts/jest/config.build.js",
     "test-build-prod": "cross-env NODE_ENV=production jest --config ./scripts/jest/config.build.js",
+    "test-dom-fixture": "cd fixtures/dom && yarn && yarn prestart && yarn test",
```

Comment:
Create a CI job for this please. Most people will not remember to run it.

Reply:
I did! It's in https://github.com/facebook/react/pull/15756/files#diff-1d37e48f9ceff6d8030570cd36286a61R167

Reply:
Oh you're right. Missed it!